### ISSUE TITLE
bug: The published builds on Docker Hub for spacebin are purely for arm64 platform

### ISSUE THREAD

**Issue by Tuxdude** [spacebin-org/spirit#448]
### What happened?

The published builds on Docker Hub for spacebin are purely for arm64 platform. Better to use multi platform images that support both amd64 and arm64 platforms.

### Minimum steps to reproduce

N/A

### Version

Any

**Comment by lukewhrit** [comment]
Latest images pushed are amd64. I've just fixed the Github action that automatically creates Docker images on release so I think we will just end up doing only amd64 images from now on, since I no longer have to build on my M1 laptop.

Also now available on Github Container Registry - https://ghcr.io/lukewhrit/spacebin

**Comment by duckfullstop** [comment]
Is there a case for re-opening this? I'd say building purely for amd64 is becoming less tenable with the upsurge in hosted arm64 platforms.

Should be pretty straightforward to use an Actions matrix or similar to build both - perhaps one of [my workflows](https://github.com/ropenttd/docker_openttd/blob/master/.github/workflows/build.yml) might be useful to you for inspiration?

**Comment by lukewhrit** [comment]
> Is there a case for re-opening this? I'd say building purely for amd64 is becoming less tenable with the upsurge in hosted arm64 platforms.
> 
> Should be pretty straightforward to use an Actions matrix or similar to build both - perhaps one of [my workflows](https://github.com/ropenttd/docker_openttd/blob/master/.github/workflows/build.yml) might be useful to you for inspiration?

Yeah, sure. You're definitely right about more arm64 systems so we can definitely add back builds to Docker. I'll reopen this and try to get it implement sooner rather than later but I'm pretty busy and Spacebin hasn't been a priority.

**Comment by lukewhrit** [comment]
Resolved in https://github.com/lukewhrit/spacebin/commit/4cbe9645f56688ed5678d859fec2402f412b466a. Fix will be available upon the next release.
